### DISCUSSION TITLE
Reverb bug ?

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1886] [Help]
Hello, 

I created an organ with GOodf and in GO when I launch it, I want to adjust the reverb. I go to the main bar, I adjust the reverb and I save the settings.

when I restart Go, the reverb parameter is back to "Max". for my parameter to be taken into account, I had to do the same manipulation 4 or 5 times. is this a bug? I did not retest with another .organ file

**Comment by kerkovits** [comment]
Isn't it related to https://github.com/GrandOrgue/grandorgue/pull/1866? That means that you need to save the setting before quitting. GO should notify (or should have notified) you on quit. @oleg68, do you flag the organ as changed after changing the release tail?
